src/dataAPI/jupyter/PythonExecutor.ts

```diff
@@ -267,6 +269,87 @@ export class PythonPandasExecutor {
         }
     }
 
+    public async textFormat(
```

Comment:
Can you use a more descriptive method name like `getTextFact` or something. Format sounds like it is just doing string formatting